src/Database/Mysql/Driver/Mysql.php

```diff
@@ -43,11 +43,19 @@ public function create(Tenant $tenant): bool
 
         event(new Events\Creating($tenant, $config, $this));
 
-        return $this->processAndDispatch(Events\Created::class, $tenant, [
-            'user'     => "CREATE USER IF NOT EXISTS `{$config['username']}`@'{$config['host']}' IDENTIFIED BY '{$config['password']}'",
-            'database' => "CREATE DATABASE `{$config['database']}`",
-            'grant'    => "GRANT ALL ON `{$config['database']}`.* TO `{$config['username']}`@'{$config['host']}'",
-        ]);
+        $qm = app(QueryManager::class);
```

Comment:
Also, let's make the variable name a bit longer / description `$queryManager` is not that long.

Reply:
That's what I get for working on this after midnight haha

Reply:
Sleep is for the weak ❤️